src/lib/features/feature-toggle/feature-toggle-service.ts

```diff
@@ -1114,9 +1114,7 @@ export class FeatureToggleService {
                 this.featureCollaboratorsReadModel.getFeatureCollaborators(
                     featureName,
                 ),
-                this.flagResolver.isEnabled('featureLinks')
-                    ? this.featureLinksReadModel.getLinks(featureName)
-                    : Promise.resolve([]),
+                Promise.resolve([]),
```

Comment:
remove this. if it breaks the return type, modify the return type to not use whatever this is used for.